gammapy/utils/scripts_test.py

```diff
@@ -0,0 +1,79 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Test if Jupyter notebooks work."""
+import os
+import sys
+import logging
+from pathlib import Path
+from pkg_resources import working_set
+from subprocess import getstatusoutput
+import yaml
+
+log = logging.getLogger(__name__)
+
+
+def get_scripts():
+    """Read `scripts.yaml` info."""
+    path = Path("examples") / "scripts.yaml"
+    with path.open() as fh:
+        return yaml.safe_load(fh)
+
+
+def requirement_missing(script):
+    """Check if one of the requirements is missing."""
+    if "requires" in script:
+        if script["requires"] is None:
+            return False
+        for package in script["requires"].split():
+            try:
+                working_set.require(package)
+            except Exception:
+                return True
+    return False
+
+
+def script_test(path):
+    """Check if example Python script is broken."""
+    log.info("   ... EXECUTING {}".format(str(path)))
+
+    cmd = "python {}".format(path)
+    status, output = getstatusoutput(cmd)
```

Comment:
`subprocess.getstatusoutput` is only there for backwards compatibility, it's use is discouraged:
https://docs.python.org/3/library/subprocess.html#legacy-shell-invocation-functions

I was mostly using `subprocess.call` and `subprocess.check_call` so far:
https://docs.python.org/3/library/subprocess.html#subprocess.check_call

But it looks like in Python 3.5 `subprocess.run` was added and it's now the first / often recommended way to run subprocesses:
https://docs.python.org/3/library/subprocess.html#using-the-subprocess-module

@Bultako - could you try to use that here?

Reply:
I've made the modif.
It works well with `subprocess.run()`